### DISCUSSION TITLE
Animation Triggers

### DISCUSSION THREAD

**Discussion by HarrisonBeachey** [uxbox/uxbox#2024] [General]
Hello!

I'm curious if there are plans to add more animation triggers (on animation end, on checkbox toggle, on screen show, etc). With a few more animation triggers the designers I work with would be interested in replacing both XD and proto.io with Penpot, which I think would be a win for everyone!

**Comment by myfunnyandy** [comment]
Hi @HarrisonBeachey

We are starting to move the discussions conversations to our new community space. It would be great if you post this same question in the [Help us improve Penpot section](https://community.penpot.app/c/give-feedback/7). That would be a better place to keep talking and better understand your needs. Having more interaction options like animation triggers is a common need more people might join and participate in this conversation.

Thanks!
